experiments/dataSources/SalesforceREST/EinsteinAnalytics/readme.md

```diff
@@ -0,0 +1,95 @@
+# Show Salesforce Einstein Analytics data in Skuid
+Einstein Analytics (EA) is a powerful data integration and analysis platform.  It does a lot of things Skuid cannot do.  But there are going to be times when you would like to incorporate an EA report into your Skuid page and use Skuid components to show the data - rather than use an Einstein Dashboard or component.  Here is how. 
+
+Some resources:
+
+- [Get a free Salesforce developer org](https://developer.salesforce.com/promotions/orgs/analytics-de) - with EA enabled and sample datasets (that we will be using in these pages)
+- [Ananlytics API doc](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_rest_resources_overview.htm) (from Salesforce Developer resources):  
+- [Analytics query language (SAQL) reference](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_saql.meta/bi_dev_guide_saql/bi_saql_intro.htm)
+
+
+## How to use this data source
+
```

Comment:
I would add a pre-req section here telling the reader to enable Einstein/sign up for an Einstein-enabled org: https://trailhead.salesforce.com/en/content/learn/projects/quick-start-einstein-analytics/sign-up-for-an-analytics-org

I kept getting errors when I tried to use the pages but couldn't figure out why. Simple in hindsight, but worth pointing out in the README.

Reply:
Eh, you **do** note this at the top of the doc.  I would say add the note here, or add a bolded note to your top doc saying _existing dev orgs will not work_. I assumed I could just use my old one.